backend/src/zimfarm_backend/common/schemas/offliners/gutenberg.py

```diff
@@ -1,135 +1,108 @@
+        title="Concurrency",
+        description="Number of concurrent threads to use",
     )
 
-    dlc = fields.Integer(
-        metadata={
-            "label": "Download Concurrency",
-            "description": (
-                "Number of parallel downloads to run (overrides concurrency)"
-            ),
-        },
-        data_key="dlc",
+    dlc: int = Field(
+        title="Download Concurrency",
+        description=("Number of parallel downloads to run (overrides concurrency)"),
+        alias="dlc",
     )
 
     # /!\ we are using a boolean flag for this while the actual option
     # expect an output folder for the ZIM files.
     # Given we can't set the output dir for regular mode, we're using this
     # flag to switch between the two and the path is set to the mount point
     # in command_for() (offliners.py)
-    one_language_one_zim = fields.Boolean(
-        truthy=[True, "/output"],
-        falsy=[False],
-        metadata={
-            "label": "Multiple ZIMs",
-            "description": "Create one ZIM per language",
-        },
-        data_key="one-language-one-zim",
+    one_language_one_zim: bool | ZIMOutputFolder = Field(
+        title="Multiple ZIMs",
+        description="Create one ZIM per language",
+        alias="one-language-one-zim",
     )
 
-    no_index = fields.Boolean(
-        truthy=[True],
-        falsy=[False],
-        metadata={
-            "label": "No Index",
-            "description": "Do not create full-text index within ZIM file",
-        },
-        data_key="no-index",
+    no_index: bool = Field(
+        title="No Index",
+        description="Do not create full-text index within ZIM file",
+        alias="no-index",
     )
 
-    title_search = fields.Boolean(
-        truthy=[True],
-        falsy=[False],
-        metadata={
-            "label": "Title search",
-            "description": "Search by title feature (⚠️ does not scale)",
-        },
-        data_key="title-search",
+    title_search: bool = Field(
+        title="Title search",
+        description="Search by title feature (⚠️ does not scale)",
+        alias="title-search",
     )
 
-    bookshelves = fields.Boolean(
-        truthy=[True],
-        falsy=[False],
-        metadata={
-            "label": "Bookshelves",
-            "description": "Browse by bookshelves feature",
-        },
+    bookshelves: bool = Field(
+        title="Bookshelves",
+        description="Browse by bookshelves feature",
     )
 
-    optimization_cache = fields.Url(
-        metadata={
-            "label": "Optimization Cache URL",
-            "description": "S3 Storage URL including credentials and bucket",
-            "secret": True,
-        },
-        data_key="optimization-cache",
+    optimization_cache: S3OptimizationCache = Field(
+        title="Optimization Cache URL",
+        description="S3 Storage URL including credentials and bucket",
+        alias="optimization-cache",
```

Comment:
Wouldn't it be simpler to override base class with a DashField like we did at https://github.com/openzim/mindtouch/blob/4772c2c2664a58b375e5490e4ff44c128e29259f/scraper/src/mindtouch2zim/ui.py#L5-L8 (but with a CamelField). Not sure there is a dash alias generator, but could be implemented in worst case scenario.